### PR TITLE
Updating Infura Endpoints -> Brave

### PR DESCRIPTION
**Pull request by ryanml**

### CODE REVIEW

File: brave/app/scripts/controllers/infura.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  /*
+   * The indicator property represents the severity of any
+   * outages or downtime. "none" indicates an "ok" status
+   */
+  async checkInfuraNetworkStatus () {
+    const response = await fetch('https://hfq2h9152m63.statuspage.io/api/v2/status.json')
+    const parsedResponse = await response.json()
+
+    this.store.updateState({
+      infuraNetworkStatus: parsedResponse,
```
Comment (ryanml) [inline brave/app/scripts/controllers/infura.js]: Screen Shot 2019-07-30 at 9 41 55 PM

File: package.json [hunk 2]
```diff
@@ -95,7 +95,7 @@
     "eth-ens-namehash": "^2.0.8",
     "eth-hd-keyring": "^1.2.2",
     "eth-json-rpc-filters": "^3.0.4",
-    "eth-json-rpc-infura": "^3.2.0",
+    "eth-json-rpc-infura": "github:brave-experiments/eth-json-rpc-infura#master",
```
Comment (ryanml) [inline package.json]: For now the project id can be hardcoded from in here for testing this PR

Comment (bbondy) [reply package.json]: I don't think you need to specify the branch name if it's master and master is the default branch

File: brave/app/scripts/controllers/infura.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+const InfuraController = require('../../../../app/scripts/controllers/infura')
+
+module.exports = class BraveInfuraController extends InfuraController {
+
+  /*
+   * The indicator property represents the severity of any
+   * outages or downtime. "none" indicates an "ok" status
+   */
+  async checkInfuraNetworkStatus () {
+    const response = await fetch('https://hfq2h9152m63.statuspage.io/api/v2/status.json')
```
Comment (bbondy) [inline brave/app/scripts/controllers/infura.js]: what's this subdomain for? `hfq2h9152m63`

Comment (ryanml) [reply brave/app/scripts/controllers/infura.js]: @bbondy that is Infura's live status.io subdomain https://infura.statuspage.io/api#status

It is intended for public use, they just don't seem to host their own directly

**Pull request merged**